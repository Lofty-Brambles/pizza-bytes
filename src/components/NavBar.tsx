import React from "react";

export default function NavBar() {
	return (
		<header>
			<img src="./pizza-logo.png" alt="logo" />
			<h1>Pizza-Bytes</h1>
		</header>
	);
}
