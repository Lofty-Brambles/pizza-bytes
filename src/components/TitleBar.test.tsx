import React from "react";
import { render } from "@testing-library/react";
import NavBar from "./TitleBar";

describe("Title bar component", () => {
	test("snapshot", () => {
		const { container } = render(<NavBar />);
		expect(container).toMatchSnapshot();
	});
});
