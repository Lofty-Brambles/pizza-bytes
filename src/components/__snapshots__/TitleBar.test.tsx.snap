// Vitest Snapshot v1

exports[`Title bar component > snapshot 1`] = `
<div>
  <header>
    <img
      alt="logo"
      src="./pizza-logo.png"
    />
    <h2>
      Pizza-Bytes!
    </h2>
  </header>
</div>
`;
